docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -13,11 +13,16 @@
   3. Navigate to the [My Apps](https://developer.bentley.com/my-apps/) page
   4. Click the **Register New** button
   5. Give your application a Name
-  6. Select the **Visualization** API
-  7. Select application type **Desktop/Mobile**
-  8. Enter **Redirect URI** `http://localhost:3000/signin-callback`
-  9. Enter **Post logout Redirect URI**: `http://localhost:3000`.
-  10. Click the **Save** button
+  6. Select the **Visualization**, **iModels**, **Reality Data**, and **Projects** APIs
```

Comment:
Based on developer.bentley.com right now, it looks like the APIs should be: Visualization, Digital Twin Management, Administration

Then for scopes:
- Visualization: imodelaccess:read
- Digital Twin Management: imodels:read, realitydata:read
- Administration: projects:read

Reply:
...though I see when editing an application entry, the API names you listed are what show up under those sub-categories. I still think we should put the API names I listed here since those show up on both the "Register New" and "Edit" pages.

Reply:
updated